tests/ballerina-test/src/test/resources/test-src/net/http/resiliency/circuit-breaker-test.bal

```diff
@@ -299,6 +396,20 @@ function handleHTTPStatusCodeErrorScenario (int counter) returns (http:Response
     return response;
 }
 
+function handleCBForceOpenScenario () returns (http:Response) {
+    return getResponse();
+}
+
+function handleCBForceCloseScenario (int requesetNo) returns (http:Response | error) {
```

Comment:
typo in `requesetNo` ?

Shall we also remove the space between the function name and brackets/parameters, and remove the brackets around the return type and make `(http:Response | error)` --> `http:Response|error` ?

Same for `handleCBForceOpenScenario`